chat-plugins/info.js

```diff
@@ -1337,6 +1337,15 @@ exports.commands = {
 		if (showAll || target === 'tiering' || target === 'tiers' || target === 'tier') {
 			buffer.push("<a href=\"https://www.smogon.com/ingame/battle/tiering-faq\">Tiering FAQ</a>");
 		}
+		if (showAll || toId(target).startsWith('lostpass') || target === 'pwreset') {
+			buffer.push("PM a member of Global Staff (@ in the Help room) to facilitate password resets.");
```

Comment:
Facilitate is kind of an awkward word to use. What about "PM a global staff member (@ and up in the Help room) for help with resetting your password."?